config/app.php

```diff
@@ -274,7 +274,7 @@
         Clarity\View\ViewServiceProvider::class,
 
         # register your provider below.
-        // Acme\Acme\AcmeServiceProvider::class,
+        Acme\Acme\AcmeServiceProvider::class,
```

Comment:
I prefer not to activate this service as it is just a dummy or just a sample sandbox class